INSTALL.md

```diff
@@ -57,8 +57,29 @@ Shogun is part of [homebrew-science](https://github.com/Homebrew/homebrew-scienc
     $ sudo brew install shogun
 
 ###Windows <a name="windows"></a>
-Shogun natively compiles under Windows using MSVC, see the [CI build](https://ci.appveyor.com/project/vigsterkr/shogun). We currently do not support a binary
-installer. If you are interested in packaging, documenting, or contributing otherwise, please contact us.
+We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC.We recommend using "Visual Studio 14 2015" which has a good support for C++11. If you do not want to install the whole IDE,you can just use MSBuild to build shogun. Running CMD as administrator with the following commands：
```

Comment:
i know i'm nitpicking a bit but could you please make spaces between the comma and fullstop. like at `using MSVC.We` or `whole IDE,you`